### PR TITLE
Add cc_proto into mixer api and config proto BUILD files.

### PR DESCRIPTION
**Pull request by qiwzhang**

### CODE REVIEW

File: mixer/v1/BUILD [hunk 1]
```diff
@@ -1,4 +1,5 @@
 load("@org_pubref_rules_protobuf//gogo:rules.bzl", "gogoslick_proto_library")
+load("@com_google_protobuf//:protobuf.bzl", "cc_proto_library")
```
Comment (lizan) [inline mixer/v1/BUILD]: why not use native ones?

File: googleapis.bzl [hunk 2]
```diff
@@ -84,10 +84,105 @@ filegroup(
     ],
 )
 
+load("@com_google_protobuf//:protobuf.bzl", "cc_proto_library")
+
+exports_files(glob(["google/**"]))
+
+cc_proto_library(
```
Comment (ayj) [inline googleapis.bzl]: Are `servicecontrol`, `service_config`, and `cloud_trace` targets used by mixerclient or proxy? A quick grep didn't turn up any meaningful results.

File: mixer/v1/BUILD (hunk 1 above)
Comment (qiwzhang) [reply mixer/v1/BUILD]: We needs two proto files; duration.proto and timestamp.proto from protobuf repo.  But protobuf repo is not using native cc_proto_library.

File: googleapis.bzl (hunk 2 above)
Comment (qiwzhang) [reply googleapis.bzl]: You are right.  Only rpc/status.proto is used. I cleaned it up.

**Comment by qiwzhang** [comment]
@ayj PTAL

**Pull request merged**